storage/object/api-cli/combining-iam-and-object-storage.mdx

```diff
@@ -0,0 +1,319 @@
+---
+meta:
+  title: Configuring access permissions for Object Storage
+  description: This page explains how to combine IAM and bucket policies to improve the configuration of your access permissions for Object Storage.
```

Comment:
we could be more straightforward: to manage access to object storage resources with granularity/  the resource/ bucket and object level